xds/internal/balancer/lrs/config.go

```diff
@@ -27,17 +27,19 @@ import (
 	"google.golang.org/grpc/xds/internal"
 )
 
-type lbConfig struct {
-	serviceconfig.LoadBalancingConfig
-	ClusterName                string
-	EdsServiceName             string
-	LrsLoadReportingServerName string
-	Locality                   *internal.LocalityID
-	ChildPolicy                *internalserviceconfig.BalancerConfig
+// LBConfig is the balancer config for lrs balancer.
+type LBConfig struct {
+	serviceconfig.LoadBalancingConfig `json:"-"`
+
+	ClusterName                string                                `json:"clusterName,omitempty"`
+	EdsServiceName             string                                `json:"edsServiceName,omitempty"`
+	LrsLoadReportingServerName string                                `json:"lrsLoadReportingServerName,omitempty"`
```

Comment:
s/Eds/EDS and s/Lrs/LRS
Same here, can the LRS prefix be stripped?

Reply:
Done